output/results.csv

```diff
@@ -0,0 +1,102 @@
+,Susceptible,Exposed,Infected,Fatalities,Recovered
```

Comment:
It seems like bad practice to track code output in a git repo - it might change with every commit and will just pollute prs, I'd .gitignore it and use sth like a Google Drive folder to share results.

Reply:
ok